osgp/protocol-adapter-iec61850/osgp-protocol-adapter-iec61850/src/main/java/org/opensmartgridplatform/adapter/protocol/iec61850/infra/messaging/DeviceResponseMessageSender.java

```diff
@@ -33,7 +34,8 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(DeviceResponseMessageSender.class);
 
-    private static final String LOG_MESSAGE_RESPONSE_MESSAGE_OF_WRONG_TYPE = "Only ProtocolResponseMessage type is expected for DeviceResponseMessageSender, received responseMessage of type {}.";
+    private static final String LOG_MESSAGE_RESPONSE_MESSAGE_OF_WRONG_TYPE = "Only ProtocolResponseMessage type is "
+            + "expected for DeviceResponseMessageSender, received responseMessage of type {}.";
     private static final String LOG_MESSAGE_JMS_EXCEPTION = "JMS Exception, closing all connections.";
     private static final String LOG_MESSAGE_BLANK_FIELD = "{} is blank.";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "LOG_MESSAGE_BLANK_FIELD" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1068)

Reply:
Done